site/components/form/SearchSelect.tsx

```diff
@@ -45,10 +47,18 @@ const SearchSelect = <T extends object>({
     isClearable = false,
     isAsync = true,
```

Comment:
Can we get rid of the isAsync version of this component if it's not being used anymore?

Reply:
done